inyoka_theme_default/templates/errors/400_csrf.html

```diff
@@ -2,36 +2,60 @@
     errors/400_csrf.html
     ~~~~~~~~~~~~~~~~~~~~
 
-    Rendered on CSRF attempts.
+    Rendered on failed CSRF attempts.
 
     :copyright: (c) 2013-2016 by the Inyoka Team, see AUTHORS for more details.
     :license: BSD, see LICENSE for more details.
 #}
 
 {% extends 'base.html' %}
 {% block title %}
-  400 - {{ super() }}
+  {{ _('Forbidden (403)') }} - {{ super() }}
```

Comment:
hm, the backend will atm return an 403 https://github.com/inyokaproject/inyoka/pull/572/files#diff-7772e09057aa2b5d81a7a1e732b0d991R1804  This is heavily inspired from Django-upstream https://github.com/django/django/blob/master/django/views/csrf.py#L145